### ISSUE TITLE
Relationship panel 'create' button not always shown for portal user

### ISSUE THREAD

**Issue by hdijkema** [espocrm/espocrm#1863]
Somehow, I don't get a '+' button for some related fields for Portal Users, while they can create records. If I select 'list' there's a button 'Create'.

It works well for related records directly under contact:

![image](https://user-images.githubusercontent.com/745693/101929808-9e03f900-3bd7-11eb-99d4-a0496815448f.png)

It doesn't one record deeper, i.e. I click on a record and want to create new related records:

![image](https://user-images.githubusercontent.com/745693/101930046-e9b6a280-3bd7-11eb-8d70-1a90257eb0c7.png)

There's no difference in the declarations, the ACLs are oke. I can add records when I click on the 3 dots, and list and create:

![image](https://user-images.githubusercontent.com/745693/101930334-4ade7600-3bd8-11eb-8439-17eb9947ff42.png)

It works well when I'm not a portal user. 

So, is there something in the code that prevents creating the 'create' +  button for portal users for related records not directly linked to the contact?

**Comment by hdijkema** [comment]
I've fixed the '+' button, by tracing what happens in javascript. The problem was that I needed to make 'nascholingstotalen' editable by the portal user. Something I'd rather not wanted to do, because it is a system generated record, but when I made it editable, the '+' sign appeared. Next I made all fields of the parent record ('nascholingstotalen'), read only for the portal user. 

However, this is I think an inconsistency. There wasn't a '+' button, but when I clicked the three dots, there was a 'create' button.
